doc/src/custom/cylc_lang.py

```diff
@@ -21,7 +21,7 @@
 
 from pygments.lexer import RegexLexer, bygroups, include
 from pygments.token import (Name, Comment, Text, Operator, String,
-                            Punctuation, Error, Keyword)
+                            Punctuation, Error, Keyword, Other)
```

Comment:
Note to reviewers: These changes are from the open PR https://github.com/metomi/rose/pull/2274.